src/test/test_dir.c

```diff
@@ -5771,22 +5771,10 @@ test_dir_networkstatus_consensus_has_ipv6(void *arg)
   /* Test the bounds for A lines in the NS consensus */
   mock_options->UseMicrodescriptors = 0;
 
-  mock_networkstatus->consensus_method = MIN_METHOD_FOR_A_LINES;
+  mock_networkstatus->consensus_method = MIN_SUPPORTED_CONSENSUS_METHOD;
   has_ipv6 = networkstatus_consensus_has_ipv6(get_options());
   tt_assert(has_ipv6);
 
-  mock_networkstatus->consensus_method = MIN_METHOD_FOR_A_LINES + 1;
-  has_ipv6 = networkstatus_consensus_has_ipv6(get_options());
-  tt_assert(has_ipv6);
-
-  mock_networkstatus->consensus_method = MIN_METHOD_FOR_A_LINES + 20;
```

Comment:
Please don't delete this one test.
Instead, replace `MIN_METHOD_FOR_A_LINES + 20` with `MIN_SUPPORTED_CONSENSUS_METHOD + 20`.

Reply:
I think MAX_SUPPORTED_CONSENSUS_METHOD (for a far-future-method) would be even better?

Going with that in 427732a5d113d190a321819f30cb90aed5b054e9